app/main/services/search_service.py

```diff
@@ -122,19 +125,21 @@ def status_for_all_indexes():
 
 def core_search_and_aggregate(index_name, doc_type, query_args, search=False, aggregations=[]):
     try:
+        mapping = app.mapping.get_mapping(index_name, doc_type)
         page_size = int(current_app.config['DM_SEARCH_PAGE_SIZE'])
         if 'idOnly' in query_args:
             page_size *= int(current_app.config['DM_ID_ONLY_SEARCH_PAGE_SIZE_MULTIPLIER'])
 
         es_search_kwargs = {'search_type': 'dfs_query_then_fetch'} if search else {}
+        q = construct_query(mapping, query_args, aggregations, page_size)
```

Comment:
a little confusing for a variable name seeing as we do have a `q` parameter on our query string...

Reply:
I think I only did that for debugging purposes, I don't really need a variable at all :)